hiku/validate/graph.py

```diff
@@ -356,7 +356,18 @@ def visit_enum(self, obj: BaseEnum) -> t.Any:
 
         self._validate_description(obj)
 
-    def visit_scalar(self, obj: t.Type[Scalar]) -> t.Any: ...
+    def visit_scalar(self, obj: t.Type[Scalar]) -> t.Any:
```

Comment:
I guess this problem common for other types as well, for example we can not have same enum and type or enum and scalar. I think we need a more general validation approach that catches duplicated type names